spring-cloud-alibaba-nacos-config/src/main/java/com/alibaba/cloud/nacos/parser/NacosDataPropertiesParser.java

```diff
@@ -16,23 +16,48 @@
 
 package com.alibaba.cloud.nacos.parser;
 
+import java.io.BufferedReader;
 import java.io.IOException;
 import java.io.StringReader;
+import java.util.Arrays;
 import java.util.Properties;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 /**
  * @author zkz
+ * @author element
  */
 public class NacosDataPropertiesParser extends AbstractNacosDataParser {
 
+	private static final Logger log = LoggerFactory
+			.getLogger(NacosDataPropertiesParser.class);
+
 	public NacosDataPropertiesParser() {
 		super("properties");
 	}
 
 	@Override
 	protected Properties doParse(String data) throws IOException {
```

Comment:
It is recommended to replace `Proterties` with `LinkedHashMap` , unnecessary conversion of data types is reduced , so you need to modify the affected code of data flow .

Reply:
> It is recommended to replace `Proterties` with `LinkedHashMap` , unnecessary conversion of data types is reduced , so you need to modify the affected code of data flow .

this PR only for fix orderly properties not for refactor